index.html

```diff
@@ -3126,110 +3126,141 @@ <h4>Semantic Interoperability</h4>
+It is also desirable to know that the contents of the JSON-LD context(s) used
+in the [=verifiable credential=] are the same when used by the [=verifier=] as
+they were when used by the [=issuer=].
         </p>
-        <p>
-It is also desirable to know that the contents of the JSON-LD context(s) used in
-the [=verifiable credential=] are the same when used by both the
-[=issuer=] and [=verifier=].
-        </p>
-        <p>
-To validate that a resource referenced by a [=verifiable credential=] is the
-same at verification time as it is at issuing time, an implementer MAY include a
-property named <code id="defn-relatedResource">relatedResource</code> that
-stores an array of objects that describe additional integrity metadata about
-each resource referenced by the [=verifiable credential=]. If
-`relatedResource` is present, there MUST be an object in the array
-for each remote resource for each context used in the verifiable credential.
-        </p>
-        <p class="issue" title="Mandatory listing of contexts in relatedResouce are under debate.">
+
+        <p class="issue" title="Mandatory listing of contexts in `relatedResource` is under debate.">
 The requirement that contexts be listed in `relatedResource` is currently being
 debated in the VCWG. This requirement might be removed in future iterations of
 the specification.
         </p>
+
         <p>
-Each object in the `relatedResource` array MUST contain the
-following: the [[URL]] to the resource named `id` and the
-<code id="defn-digestSRI">digestSRI</code> information for the resource
-constructed using the method specified in
-<a href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource Integrity</a>.
+To extend integrity protection to a related resource, an [=issuer=] of a
+[=verifiable credential=] MAY include the `relatedResource` property:
         </p>
+
+        <dl>
+          <dt id="defn-relatedResource">relatedResource</dt>
+          <dd>
+The value of the `relatedResource` property MUST be one or more objects of the
+following form:
+            <table class="simple">
+              <thead>
+                <th>Property</th>
+                <th>Description</th>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>`id`</td>
+                  <td>
+The identifier for the resource is REQUIRED and conforms to the format defined
+in Section [[[#identifiers]]]. The value MUST be unique among the list of
+related resource objects.
+                  </td>
+                </tr>
+                <tr>
+                  <td>`mediaType`</td>
+                  <td>
+An OPTIONAL valid media type as listed in the
+<a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+IANA Media Types</a> registry.
+                  </td>
+                </tr>
+                <tr>
+                  <td>`digestSRI`</td>
+                  <td>
+One or more cryptographic digests, as defined in [[[SRI]]].
```

Comment:
The reference to the SRI specification may be a bit problematic; that specification describes a general framework for HTML, and we use only one part of it, namely the [integrity attribute](https://www.w3.org/TR/SRI/#the-integrity-attribute). To be even more precise, we use the BNF definition therein. 

Note that we have, in our spec, a precise definition for an [SRI string definition](https://www.w3.org/TR/SRI/#the-integrity-attribute) which does that precise reference. Wouldn't it be better to refer directly to that section?

Reply:
Fixed in 4e7d4b366e564d9d158f082a1f58e8f8e7ed4a3a.